src/content/docs/infrastructure/host-integrations/host-integrations-list/lightptd-integration.mdx

```diff
@@ -0,0 +1,152 @@
+
+1. Go to lighttpd config file path
+```
+cd /etc/lighttpd/
+```
+2. Open `lighttpd.conf` file to edit
+```
+sudo nano lighttpd.conf
+```
+3. Add `mod_status` on your Lighttpd servers.
+4. Add `mod_status` inside `server.modules` object.
+5. Add status.status-url = `/server-status` after server.modules
+6. Update `server.document-root` with the lighttpd file path `/var/www/html/lighttpd-webpage`
+7. Update `server.port` on which you are running lighttpd server.port = 9880
+8. Run `127.0.0.1:9880/server-status?auto` or  `127.0.0.1:9880/server-status?format=plain` on the browser to check server related metrics.
+</Step>
+
+
+<Step>
+
+## Nri-Flex configuration
+
+First, follow the [installation steps](https://github.com/newrelic/nri-flex/blob/master/docs/basic-tutorial.md) of nri-flex from [nri-flex repository](https://github.com/newrelic/nri-flex)
+
+Once you have installed the infrastructure agent on your host, nri-flex binary is also installed with it.
+
+1. Create a flex configuration file
+Path:
+```
+/etc/newrelic-infra/integrations.d
+```
+
+2. Create a file with the name `lighttpd-http-config.yml`
+
+Note: A generic file name could be &ltFILENAME&gt-config.yml
+
+3. Use add the following configuration to your `lighttpd-http-config.yml` file. 
+
+Sample configuration file: https://github.com/newrelic/nri-flex/blob/master/examples/lighttpd-http-example.yml
+
+It's important to not that you can replace `base_url` and `url` with your application url (Please refer to url documentation for more understanding) and also change url in the `commands` attribute. You can also convert the metrics format which you get through url into regex and add expressions in regex_matches. EVENT_TYPE is also adjustable. 
+
+Once your configuration file is updated, it will look like this:
+
+```
+---
+integrations:
+  - name: nri-flex
+    # interval: 30s
+    config:
+      name: lighttpdFlex
+      global:
+        base_url: http://lighttpd.net/
+      apis:
+        - event_type: status
+          url: status?format=plain
+        - event_type: &lTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+          commands:
+            - cache: status?format=plain
+              split_by: ": "
+          snake_to_camel: true
+
+
+  - name: nri-flex
+    interval: 10s
+    # timeout: 5s
+    config:
+      name: test
+      apis:
+        - event_type: &ltTHE_PLACE_WHERE_METRICS_ARE_CAPTURED&gt
+          commands:
+            - run: curl 'http://lighttpd.net/status?auto'
+              split_by: ": "
+              regex_matches:
+                - expression: (\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)\n(\w+ \w+|\w+):(\N+)
+```
+
+</Step>
+
+<Step>
+
+### Restart the New Relic infra agent and lighttpd service.
+
+```
+sudo systemctl restart lighttpd.service && sudo systemctl restart newrelic-infra.service
+```
+</Step>
+
+<Step>
+###  Query your data using NRQL
+
```

Comment:
Execute the below sample query on the New Relic's query builder to see the metrics which are captured on one.newrelic.com